### PR TITLE
Ntp revamp

### PR DESCRIPTION
**Pull request by vxav**
I did some testing of [#89](https://github.com/giantswarm/cluster-cloud-director/pull/89) and I propose this implementation for NTP

Tested in Grenoble.

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] Update Lastpass values if required.

### CODE REVIEW

File: helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml [hunk 1]
```diff
@@ -90,6 +90,7 @@ spec:
           {{- end }}
     files:
       {{- include "sshFiles" . | nindent 6}}
+      {{- include "ntpFiles" . | nindent 6}}
```
Comment (erkanerol) [inline helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml:93]: Do we need this for kubeadmconfigtemplate too?

Comment (vxav) [reply helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml:93]: We also want to configure ntp on the worker nodes.

Comment (erkanerol) [reply helm/cluster-cloud-director/templates/kubeadmcontrolplane.yaml:93]: sorry. i missed this line https://github.com/giantswarm/cluster-cloud-director/pull/101/files#diff-7188efb4466b71c05057ba8992c443f8c9ab6bc2e1d67049a72f6a323aae2bdcR128

File: helm/cluster-cloud-director/values.yaml [hunk 2]
```diff
@@ -95,6 +95,10 @@ nodePools: {}
   #   class: default  # Node classes name (defined above)
   #   replicas: 2
 
+ntp:  # ntp pools and servers chrony config
+  servers: []
```
Comment (erkanerol) [inline helm/cluster-cloud-director/values.yaml:99]: Can we add some default values here? is it possible?

Comment (vxav) [reply helm/cluster-cloud-director/values.yaml:99]: If we don't set anything, the default `chrony` pools are applied (currently what happens). 
The rest of the config will be the same as I just matched it.

```
pool ntp.ubuntu.com        iburst maxsources 4
pool 0.ubuntu.pool.ntp.org iburst maxsources 1
pool 1.ubuntu.pool.ntp.org iburst maxsources 1
pool 2.ubuntu.pool.ntp.org iburst maxsources 2
```

**Review by erkanerol — APPROVED** [review]

**Pull request merged**